lib/exabgp/bgp/message/update/nlri/evpn/mac.py

```diff
@@ -42,8 +42,12 @@ class MAC (EVPN):
 	NAME = "MAC/IP advertisement"
 	SHORT_NAME = "MACAdv"
 
-	def __init__ (self, rd, esi, etag, mac, maclen, label, ip, packed=None):
-		EVPN.__init__(self, packed)
+	def __init__ (self, rd, esi, etag, mac, maclen, label,ip,packed=None,nexthop=None,action=None,addpath=None):
+		EVPN.__init__(self,packed,nexthop,action,addpath)
+		assert(isinstance(rd, RouteDistinguisher))
```

Comment:
As you wish. I thought they were helpful safeguard and that the impact could be zero at runtime by starting exabgp with "python -0".